scripts/publish-deb.sh

```diff
@@ -12,7 +12,7 @@ DEBS3='deb-s3 upload '\
 '--s3-region=us-west-2 '\
 '--bucket packages.o1test.net '\
 '--preserve-versions '\
-'--lock '\
+'--no-lock '\
```

Comment:
NOOOOOOO! Delete the lockfile as described in the comment above, don't remove the lock flag!!